src/jit/emitxarch.cpp

```diff
@@ -5535,15 +5535,28 @@ static bool isSseShift(instruction ins)
     }
 }
 
-static bool isSSEExtract(instruction ins)
+//------------------------------------------------------------------------
+// IsDstSrcImmAvxInstruction: check if instruction has RM R I format
+// for all encodings: EVEX, VEX and legacy SSE
+//
+// Arguments:
+//    instruction -- processor instruction to check
+//
+// Return Value:
+//    true if instruction has RRI format
```

Comment:
The return vale comment should be modified as well.

Reply:
Will fix. Question as above: could we postpone it to my next PR?